selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/requestfactory/TestDefaultUserHttpJsonRequestFactory.java

```diff
@@ -10,22 +10,37 @@
  */
 package org.eclipse.che.selenium.core.requestfactory;
 
+import static java.lang.String.format;
+
 import com.google.inject.Inject;
-import com.google.inject.Provider;
 import com.google.inject.Singleton;
-import org.eclipse.che.selenium.core.user.DefaultTestUser;
+import org.eclipse.che.selenium.core.client.TestAuthServiceClient;
+import org.eclipse.che.selenium.core.user.TestUser;
 
-/** @author Dmytro Nochevnov */
+/**
+ * @author Dmytro Nochevnov
+ * @author Anton Korneta
+ */
 @Singleton
 public class TestDefaultUserHttpJsonRequestFactory extends TestHttpJsonRequestFactory {
```

Comment:
This class looks like redundant after default test user is replaced by regular test user, doesn't it?

Reply:
yes you are right